modules/installation-osp-deleting-bootstrap-resources.adoc

```diff
@@ -45,11 +45,12 @@ Delete the bootstrap resources that you no longer need.
 
 . On a command line, run the playbook:
 +
+[source,terminal]
 ----
 $ ansible-playbook -i inventory.yaml down-03_bootstrap.yaml
 ----
 
 The bootstrap port, server, and floating IP address are deleted.
 
 [WARNING]
-If you have not disabled the bootstrap Ignition file URL, do so now.
\ No newline at end of file
+If you have not disabled the bootstrap Ignition file URL, do so now.
```

Comment:
If you did not disable the bootstrap Ignition file URL earlier, do so now. 

?